logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt

```diff
@@ -307,6 +310,24 @@ internal class ConversationMapperImpl(
         }
     }
 
+    private fun receiptFromApiToDaoModel(receiptMode: ReceiptMode?): ConversationEntity.ReceiptMode = when (receiptMode) {
```

Comment:
```suggestion
    private fun receiptModeFromApiToDaoModel(receiptMode: ReceiptMode?): ConversationEntity.ReceiptMode = when (receiptMode) {
```

Include full name to avoid confusion? Same thing applies for other mapping methods

Reply:
This mapper have been moved to a proper mapper class and it is handled (naming + tests) in another PR, [can check it here](https://github.com/wireapp/kalium/pull/1218).